README.md

```diff
@@ -1 +1,192 @@
-# WebpackBoiler
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+ 
+ 
+
+  <h3><b>Leader board</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Leader board] <a name="about-project"></a>
+
+Describe your project in 1 or 2 sentences.
+
+**[Leader board]** is a website that displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+## 🛠 Built With <a name="built-with"></a>
+
+- Html
+- CSS
+- Javascript
+
+### Tech Stack <a name="tech-stack"></a>
+
+Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS, JAVASCRIPT</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+Describe between 1-3 key features of the application.
+
+- **[Single page Application]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]( https://frankmugagga.github.io/Leaderboard/dist/ )
+
+
+
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  https://github.com/FrankMugagga/Leaderboard.git
+```
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd Leaderboard
+```
```

Comment:
- [ ] [OPTIONAL] The command you have provided is specific to project setup steps. Kindly consider the use of installation commands for the dependencies. Since this is a webpack project, you may consider using `npm install` ☺️.